server/lib/genome/importers/file_importers/chembl.rb

```diff
@@ -0,0 +1,81 @@
+module Genome; module Importers; module FileImporters; module Chembl;
+  class Importer < Genome::Importers::Base
+    attr_reader :file_path
+    attr_accessor :chembl_data
+    def initialize(file_path)
+      @file_path = file_path
+      @source_db_name = 'ChEMBL'
+    end
+
+    def create_claims
+      query_chembl_db
+      create_interaction_claims
+    end
+
+    def create_new_source
+        @source ||= Source.create(
+          {
+            base_url: 'https://chembl.gitbook.io/chembl-interface-documentation/downloads',
+            site_url: 'https://www.ebi.ac.uk/chembl/',
+            citation: 'ChEMBL: towards direct deposition of bioassay data. Mendez D, Gaulton A, Bento AP, Chambers J, De Veij M, Félix E, Magariños MP, Mosquera JF, Mutowo P, Nowotka M, Gordillo-Marañón M, Hunter F, Junco L, Mugumbate G, Rodriguez-Lopez M, Atkinson F, Bosc N, Radoux CJ, Segura-Cabrera A, Hersey A, Leach AR. Nucleic Acids Res. 2019; 47(D1):D930-D940. doi: 10.1093/nar/gky1075',
+            source_db_version: '09-Mar-2022',
+            source_trust_level_id: SourceTrustLevel.EXPERT_CURATED,
+            source_db_name: source_db_name,
+            full_name: 'The ChEMBL Bioactivity Database',
+            license: 'Creative Commons Attribution-Share Alike 3.0 Unported License',
+            license_link: 'https://chembl.gitbook.io/chembl-interface-documentation/about'
+          }
+        )
+        @source.source_types << SourceType.find_by(type: 'interaction')
+        @source.save
+    end
+
+    def query_chembl_db
+        puts File.absolute_path(file_path)
+        db = SQLite3::Database.open file_path
```

Comment:
Looks like this is a new gem -- the Gemfile and Gemfile.lock need to get included in this PR too